src/configure_config.cmake

```diff
@@ -138,3 +138,6 @@ endif()
 if (NOT ENABLE_EXTERNAL_OPENSSL)
     set(USE_BORINGSSL 1)
 endif ()
+if (ARCH_PPC64LE)
```

Comment:
I don't understand this.

Reply:
This macro is used to identify the PPC architecture and use the crc32 specific function for power.

Reply:
It's done. Removed the macro and used the compiler provided macro.